tilequeue/command.py

```diff
@@ -457,7 +457,7 @@ def _parse_postprocess_resources(post_process_item, cfg_path):
         try:
             fn = resolve(init_fn_name)
 
-        except:
+        except Exception:
```

Comment:
This feels really un-Pythonic to me, but not strongly enough to say we should ignore flake.